washee_box/Readme.md

```diff
@@ -6,14 +6,161 @@ logging errors
 communication with washee-web (potentially)
 communication with input devices like fingerprint scanner,nfc,wifi-to mobileapp, water sensor, maintennance sensors...
 
+
+# Startup procedure on hardware
+## if not created create virtual environment
```

Comment:
What directory should i start in? Should i ssh? Have Mate or something else installed already? What Pi do you use?

Otherwise, a good description, will try it when i get my own pi up and running!